apps/studio/components/interfaces/Database/Roles/CreateRolePanel.tsx

```diff
@@ -87,8 +87,8 @@ const CreateRolePanel = ({ visible, onClose }: CreateRolePanelProps) => {
           <FormActions
             form={formId}
             isSubmitting={isCreating}
-            hasChanges={form.formState.isDirty}
-            handleReset={() => form.reset()}
+            hasChanges={true}
```

Comment:
quick question - this doesnt look right - it should still be `form.formState.isDirty` right?

Reply:
`hasChanges={true}` is set to `true ` as to enable form action buttons, and if the user clicks on 'Save' with an empty value, form validation is triggered, hence I feel this is valid as the user will be blocked until they enter a valid value.